editor/project_manager.cpp

```diff
@@ -1088,6 +1090,10 @@ void ProjectManager::_full_convert_button_pressed() {
 	ask_full_convert_dialog->get_cancel_button()->grab_focus();
 }
 
+void ProjectManager::_migration_guide_button_pressed() {
+	OS::get_singleton()->shell_open("https://docs.godotengine.org/en/stable/tutorials/migrating/");
```

Comment:
This should probably be a constant (well, `#define`), and combine with the existing definition in `version_generated.gen.h`. For example:

https://github.com/godotengine/godot/blob/46c495ca21f40f57a7fb9c7cde6143738f1652d4/editor/editor_help.cpp#L70